hadoop-ozone/tools/src/test/java/org/apache/hadoop/ozone/audit/parser/TestAuditParser.java

```diff
@@ -42,9 +42,9 @@
 import java.util.List;
 
 /**
- * Tests GenerateOzoneRequiredConfigurations.
+ * Tests TestAuditParser.
  */
-public class TestAuditParser {
+public class AuditParser {
```

Comment:
Hi @pingsutw 
The class name should be TestAuditParser.
Comment for the class should be Test AuditParser.

Reply:
Sorry for my mistake, I have updated patch